packages/kmod-5.10-nvidia/kmod-5.10-nvidia.spec

```diff
@@ -89,6 +97,15 @@ sed \
   -e "s|__PREFIX__|%{_cross_prefix}|" %{S:200} > nvidia.conf
 install -p -m 0644 nvidia.conf %{buildroot}%{_cross_tmpfilesdir}
 
+# disable-gsp
+ install -p -m 0644 \
+   disable-gsp.service \
+   %{buildroot}%{_cross_unitdir}
+install -d %{buildroot}%{_cross_factorydir}/etc/modprobe.d
```

Comment:
This is OK, but putting it in `/usr/share/factory` invites the misconception that `tmpfiles` is supposed to manage it. It could just as easily go in `%{_cross_datadir}/nvidia/modprobe.d` and avoid that confusion.